quizbox.css

```diff
@@ -0,0 +1,93 @@
+* {
```

Comment:
you can move the reset code and other common code to some common css file(maybe to styles.css file)